packages/backend/src/utils/parsers/quadlet-unit-parser.ts

```diff
@@ -45,6 +45,15 @@ export class QuadletUnitParser extends Parser<string, Quadlet> {
     return randomUUID();
   }
 
+  // https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Service%20Templates
+  protected isTemplate(): boolean {
```

Comment:
Seems to be the same as in packages/backend/src/utils/parsers/quadlet-dryrun-parser.ts
My suggesion would be to refactor in another utils lib

Reply:
Small difference, one check for .service the other for .container, .image etc.

We could factorise, but IMO small enough for keeping both